src/apis/price-updater.ts

```diff
@@ -1,20 +1,43 @@
 import axios from "axios";
+import * as z from "zod";
 
 import { CurrencySymbol } from "src/utils/currencies";
 
 // domain
 import { FiatExchangeRates, Token } from "src/domain/hermez";
 
+// persistence
```

Comment:
Do you mean to remove the comments and the empty lines or just the empty lines?

Reply:
Jus the empty lines, I love comments haha

Reply:
I've investigated and concluded that there's no way to enforce this with either prettier or an eslint rule, so it's difficult to maintain consistency in the future.

For now, I've manually removed all the spaces in this [commit](https://github.com/hermeznetwork/wallet-ui/pull/663/commits/ee3c0d8b963dd6029182b3b00bc107a82076a5a3), but I was in the branch of slice 4 by mistake. Since it's not an urgent change, is it ok to leave the change for the PR of the slice 4?

Reply:
Sure, let's leave it like this. Yup, this is hard to enforce, but if prettier also doesn't enforce a new line between imports, then we can use PRs to maintain the coding style as well